packages/xrpl/src/Wallet/fundWallet.ts

```diff
@@ -20,15 +20,103 @@ const INTERVAL_SECONDS = 1
 const MAX_ATTEMPTS = 20
 
 /**
- * Generates a random wallet with some amount of XRP (usually 1000 XRP).
+ * The fundWallet() method is used to send an amount of XRP (usually 1000) to a new (randomly generated)
+ * or existing XRP Ledger wallet.
  *
  * @example
- * ```typescript
+ *
+ * Example 1: Fund a manually random generated wallet
+ * const { Client, Wallet } = require('xrpl')
+ *
+ * const client = new Client('wss://s.altnet.rippletest.net:51233')
+ *
+ * async function fundRandomWallet() {
+ *   const newWallet = new Wallet()
+ *   const walletData = newWallet.generate()
+ *
+ *   try {
+ *     const { balance } = await client.fundWallet(walletData.publicKey, { xrpAmount: '10' })
+ *     console.log(`Sent 10 XRP to wallet: ${wallet.publicKey}, balance: ${balance} XRP`)
+ *   } catch (error) {
+ *     console.error(`Failed to fund wallet: ${error}`)
+ *   }
+ * }
+ *
+ * fundRandomWallet()
+ *
+ * In this example, we use the generate() method of the Wallet class to create a new random wallet.
+ * The walletData object returned by the generate() method contains the publicKey and secret values for the new wallet.
+ *
+ * The fundWallet() method of the Client class is then called with the publicKey value of the new wallet as the first
+ * argument, and an object that specifies the amount of XRP to send in the xrpAmount parameter. In this case, the
+ * xrpAmount parameter is set to '10', which is equivalent to 10 XRP.
+ *
+ * If the transaction is successful, the function logs a message to the console with the wallet public key and balance of the
+ * new wallet. If there's an error funding XRP to the wallet, the function logs an error message to the console using
+ * console.error().
+ *
```

Comment:
Similar to previous, this is the happy path we want them to follow & updated the description to include when you'd want to use the optional parameters. For this function in particular I think it's ok not to have the error handling because if the account doesn't get funded, everything else will break almost guaranteed.

```suggestion
 * Example 1: Fund a randomly generated wallet
 * const { Client, Wallet } = require('xrpl')
 *
 * const client = new Client('wss://s.altnet.rippletest.net:51233')
 * await client.connect()
 * const wallet = await client.fundWallet()
 *
 * Under the hood, this will use `Wallet.generate()` to create a new random wallet, then ask a testnet faucet
 * To send it XRP on ledger to make it a real account. If you'd like, you can also re-fill an existing account by
 * Passing in a Wallet you already have. 
```